### ISSUE TITLE
TODAO

### ISSUE THREAD

**Issue by usama9500** [commons-stack/commonsprize#32]
## Tell us about the community that you are nominating
Facilitating the conversion to electric vehicles of small public transportation operators in the Philippines

## What public goods does this community support or will they support in the future?
Public transportation

## Who are the people, DAOs and other organizations already part of this community?
In the Philippines, public transportation is currently regulated by the city.  There are many organizations with small area concessions in a single municipality

## Why do you think this community needs a Commons?
Many of Elinor Ostrum's classic case studies came from the Philippines.  From my observations in the past 6 years, they continue to organize themselves like a commons today

## What other resources do you have that will make your Commons deployment a success? 
The bootstrapped flywheel I am envisioning stems from small devices that we can install to reduce ongoing fuel costs in their existing vehicles.  If those fuel savings are pooled, they can be used to systematically convert their oldest vehicles towards electric.  If the organization is sufficiently large, they could also qualify for better financing packages.I am basically by myself here in the Philippines.

## Do you have an idea for the name of this Commons?
TODAO - (TODA is the current name for the thousands of Tricylcle Operators/Drivers Associations across the country)

## Submitted by [Discord handle or Twitter handle]
Discord: graial#0787	
Twitter: @in_economy &  @banal_na_graial

**Comment by graial** [comment]
Greetings!
Adding a post here to make sure I get a notification about any comments on my submission.
Thanks for checking it out!

**Comment by LACHelp** [comment]
Thank you for thinking of our device Allan!  I would like to make some corrections if I may...
The inventor is not Filipino, however, the device is made by disabled Filipinos via this NGO: http://fti2009.com
Also, the emissions reduced are Philippine gov't-verified at: CO2 = 16% down  &  HCs = 65% down
Our average trike driver income is increased by 10% due to his lower fuel expenses.
Here is another video by an NGO that did a project on 60 trikes in Manila:  https://www.youtube.com/watch?v=w-d7N7UOpqo
Thanks again and best wishes on your challenge!

**Comment by graial** [comment]
` Our average trike driver income is increased by 10% due to his lower fuel expenses. `

As I remember it, the device is cheap enough to deliver a very healthy payback period (less than 3 months)!  So it could unlock some very substantial financial momentum!

**Comment by graial** [comment]
`Our average trike driver income is increased by 10% due to his lower fuel expenses.`

As I remember it, the device is also cheap enough to deliver a very healthy payback period (less than 3 months)! So it could unlock some very substantial financial momentum!

**Comment by graial** [comment]
Our average trike driver income is increased by 10% due to his lower fuel expenses.

As I remember it, the device is also cheap enough to deliver a very healthy payback period (less than 3 months)! So it could unlock some very substantial financial momentum!

Thanks so much for the corrections.  I had to throw this video together really quickly and its such a short overview so for sure some details that I could've explained better

**Comment by LACHelp** [comment]
Yes, you are correct, though in actual tests payback usually occurs in 2 
months or less.  :-)

**Comment by LACHelp** [comment]
No worries on the video!  Is there a link to it wherein an account is 
not necessary to view it?

**Comment by graial** [comment]
There's just the one video that's linked above.
But we had an online meeting where each of the submissions did a quick presentation here's the clip where I presented TODAO
https://youtu.be/_wikGRhQfRE?t=2691

**Comment by Jose5048** [comment]
Wonderful project IMO but and it is relevant for all types of local communities urban and rural and will really take off with swappable batteries. I would try to model this, create a digital twin of this initiative. This like so many other projecst here deserves to be independtly funded, once you deplo a PoC.

**Comment by LACHelp** [comment]
Thank you for your comment Jose, but unfortunately even the Int'l Energy 
Agency predicts that EV deployment will be ~/at best/~ 10% of the 2 
billion vehicles in the world by 2030.   This of course leaves 1.8 
billion polluting vehicles on planet Earth. Earth doesn't have time to 
wait for EVs -- it needs a fast solution NOW.

**Comment by Jose5048** [comment]
_"The findings in the field of the energy consumption indicate that e-scooters could travel 128 km with 1 kW/h of energy, whereas a fossil fuel car can travel less than 1.6 km and the best-in-class e-cars can travel 6.4 km with the same amount of energy [[37](https://www.mdpi.com/2076-3417/11/13/5851/htm#B37-applsci-11-05851)]. The development of e-micromobility might change the overall energy consumption in the mobility sector. The findings in Pittsburg indicated that **the modal shift from conventional motorized modes of transport to e-micromobility for trips in the range of 5–8 km will decrease the energy demands of mobility by 50 percent** [[35](https://www.mdpi.com/2076-3417/11/13/5851/htm#B35-applsci-11-05851)]. The CO2 emissions of e-scooters are approximately between 125–131 g per passenger per km, including 50 percent from the manufacturing process and materials, and 43 percent from the collection process for overnight charging. Therefore, their lifespan has an important environmental impact, because of the considerable emissions involved in their production process"_
https://journals.sagepub.com/doi/full/10.1177/0269094220968231 .
**This is for urban transport.**

**Comment by Jose5048** [comment]
While the impact of micromobility in urban areas is pretty clear, its a bit more challenging for rural areas. But the solutions are there and being implemented. **_The availability of transport has a cost benefit factor of 3 and rural incomes rose by 8% per annum - a potential doubling of income in 9 years._**
**[Unequal futures of rural mobility: Challenges for a “Smart Countryside](https://journals.sagepub.com/doi/full/10.1177/0269094220968231)**”
But swappable batteries 
[Powering Electric Two- and Three-Wheelers with Rural Minigrids in Nigeria and India](https://rmi.org/powering-electric-two-wheelers-with-rural-minigrids-in-nigeria-and-india/)

**Comment by Jose5048** [comment]
_"The real point is micromobility is important not just for its impact on the environment **but also for boosting local economies** in addition to being environmentally friendly"_

**Comment by graial** [comment]
Statistics aside, I think we would welcome attempts to create a digital twin or other model on TODAO.
That was a big reason we chose to participate in the commons prize in the first place.  To develop the relationships that would help us drive an initiative like this forward.

**Comment by LACHelp** [comment]
Jose,

I have no doubt that your data is correct, however, you do not mention 
that the Int'l Energy Agency predicts that EV deployment will be ~/at 
best/~ 10% of the 2 billion vehicles in the world by 2030.

What this means is that while your data is admirable and logical, it 
cannot happen soon enough, in large enough numbers, to make a serious 
dent in the current level of emissions produced by vehicles, yet this is 
what NEEDS to be addressed immediately.

**Comment by usama9500** [comment]
The issue has been closed as the first round has concluded! Awesome work @graial 🙌
